docs/Using Any API/multi-variable-responses.md

```diff
@@ -2,64 +2,109 @@
-
 **Table of Contents**
 
-+ [MultiWord](#multiword)
-+ [Choosing an Oracle and JobId](#choosing-an-oracle-and-jobid)
-+ [Make an Existing Job Request](#make-an-existing-job-request)
+- [MultiWord](#multiword)
+- [Setting the LINK token address, Oracle, and JobId](#setting-the-link-token-address-oracle-and-jobid)
+- [Make an Existing Job Request](#make-an-existing-job-request)
 
 ## MultiWord
 
-To consume an API with multiple responses, your contract should inherit from [ChainlinkClient](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/ChainlinkClient.sol). This contract exposes a struct called `Chainlink.Request`, which your contract should use to build the API request. The request should include the following:
+This example shows how to:
+
+- Fetch several responses in one single call.
+
+[Cryptocompare GET /data/price/ API](https://min-api.cryptocompare.com/documentation?key=Price&cat=SingleSymbolPriceEndpoint) returns the current price of any cryptocurrency in any other currency that you need. To check the response, you can directly paste the following URL in your browser `https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=BTC` or run this command in your terminal:
+
+```curl
+curl -X 'GET' \
+  'https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=BTC' \
+  -H 'accept: application/json'
+```
+
+The response should be similar to the following:
+
+```json
+{
+  "BTC": 0.07297
+}
+```
+
+The request above shows how to get the price of _ETH_ against _BTC_. Now let say we want the price of _ETH_ against several currencies: _BTC_, _USD_, and _EUR_. Our contract will have to support receiving multiple responses.
+To consume an API with multiple responses, your contract should inherit from [ChainlinkClient](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/ChainlinkClient.sol). This contract exposes a struct called `Chainlink.Request`, which your contract should use to build the API request. The request should include the following parameters:
 
+- Link token address
 - Oracle address
 - Job id
 - Request fee
 - Task parameters
 - Callback function signature
 
->❗️ Note on Funding Contracts
+> ❗️ Note on Funding Contracts
 >
 > Making a GET request will fail unless your deployed contract has enough LINK to pay for it. **Learn how to [Acquire testnet LINK](../acquire-link/) and [Fund your contract](../fund-your-contract/)**.
->
+
+Assume that a user wants to obtain the ETH price quoted against three different currencies: _BTC_ , _USD_ and _EUR_. If they use only a single-word job (cf. [Single Word Response](/docs/single-word-response/)), it would require three different requests. To make that more efficient, they can use multi-word responses to do it all in a single request as shown in the following example:
```

Comment:
```suggestion
Assume that a user wants to obtain the ETH price quoted against three different currencies: _BTC_ , _USD_ and _EUR_. If they use only a single-word job, it would require three different requests. For a comparison, see the [Single Word Response](/docs/single-word-response/) example. To make these requests more efficient, use multi-word responses to do it all in a single request as shown in the following example:
```